.changeset/cyan-icons-rest.md

```diff
@@ -0,0 +1,14 @@
+---
+'@backstage/plugin-explore-backend': patch
+---
+
+Add support for the new backend system.
+
+A new backend plugin for the explore backend
+was added and exported as `default`.
+
+You can use it with the new backend system like
+
+```ts title="packages/backend/src/index.ts"
```

Comment:
Not sure this is gonna show up anywhere, but let's try it 😁

Reply:
seems like it doesn't